app/services/google_service/client_service.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module GoogleService
+  class ClientService
+    require 'httparty'
+    require 'colorize'
+
+    USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) '\
+                'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36'
+
+    def initialize(keyword, lang = 'en')
+      escaped_keyword = CGI.escape(keyword)
+      @uri = URI("https://google.com/search?q=#{escaped_keyword}&hl=#{lang}&gl=#{lang}")
+    end
+
+    def query_result
```

Comment:
Based on out convention, this method name should be `call` https://nimblehq.co/compass/development/code-conventions/ruby-on-rails/#services

Reply:
Is call a real convention or just a part of the example? 
If it's a valid convention, I would submit a RFC in Compass so that we can add 1 or 2 lines to clarify what is convention from what is purely example :)

Reply:
PS: Rename done, convention or not, it's much cleaner :D